### PR TITLE
Adding introduction of Smart Speaker

### PR DESCRIPTION
**Pull request by jessizheng**
Bug Fixes #67 

**Description about the pull request**
- recommit the files by using 'Git' instead of 'SVN'

**Review**
@MarkDing 
@Jim-tech  
@ChengYuan-CY 
@hoo-way

### CODE REVIEW

**Comment by jessizheng** [comment]
Hi all,

I apologized for making some error in last version. I recommitted the work and it should be readable now. In order to reduce the work, I reassign these articles to those who wrote it . Please check the list below :

@MarkDing : 
- Tmall Genie Smart Speaker ecosystem Introduction and Setup
- Tmall Genie control the Zigbee devices via cloud with AliOS   

@Jim-tech : 
- Tmall Genie control the Zigbee devices via cloud with Micrium OS 
- Cost of Development Analyses and Ecosystem Comparisons   

@ChengYuan-CY :
- Alexa control the Bluetooth Mesh devices via cloud 
- China Smart Speaker Ecosystem Investigation 
- Tmall Genie control the Bluetooth Mesh Devices with the Built-in Hub
- Smart Speaker Solution Summary and extend the capability of EFR32 for the ecosystems support   

@hoo-way : 
- The US Smart Speaker Ecosystem Investigation 
- Tmall Genie control the WIFI devices via cloud 

To remind, **Alexa Smart Speaker ecosystem Introduction and Setup** and **Alexa control the Zigbee devices with the built-in hub** haven't been written yet. please let me know if you have any advice. Thank you in advance for your helping.

Best Regards
Jessica

File: common/China-Smart-Speaker-Ecosystem-Investigation.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+- [4. IoT设备接入方式对比](#4-iot设备接入方式对比)
+    - [4.1. AliGenie IoT开放平台](#41-aligenie-iot开放平台)
+        - [4.1.1. 云云接入](#411-云云接入)
+        - [4.1.2. 直接接入](#412-直接接入)
+            - [4.1.2.1. WIFI设备接入](#4121-wifi设备接入)
+            - [4.1.2.2. 蓝牙Mesh设备接入](#4122-蓝牙mesh设备接入)
+            - [4.1.2.3. 低功耗蓝牙设备接入](#4123-低功耗蓝牙设备接入)
+    - [4.2. 小米IoT开放平台](#42-小米iot开放平台)
+        - [4.2.1. 设备直接接入](#421-设备直接接入)
+            - [](#)
```
Comment (ChengYuan-CY) [inline common/China-Smart-Speaker-Ecosystem-Investigation.md:32]: 4.2.1.1 is missing

File: common/China-Smart-Speaker-Ecosystem-Investigation.md [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+<!-- /TOC -->
+</details>
+
+
+
+# 1. 概述
+
+智能音箱作为智慧家庭的一个重要控制入口，受到各个科技大厂的青睐。相比于传统的仅支持音乐播放功能的音箱而言，智能音箱加入了语音对话，语义理解，智能家居设备控制等功能。
+
+为了对目前国内市场上所流行的智能音箱有一个更为深入的了解，我们通过这一个confluence page来对其作一个分析对比。我们选取了目前国内市场上最为流行的三款智能音箱设备来作对比，它们分别是阿里巴巴的天猫精灵，小米的小爱同学以及百度的小度。因为我们主要关注于智能音箱来控制IoT智能家居设备，因此该文章也主要侧重于智能音箱的智能系统，IoT设备接入方式等几个方面。
```
Comment (ChengYuan-CY) [inline common/China-Smart-Speaker-Ecosystem-Investigation.md:48]: 我们通过这一个confluence page来对其作一个分析对比
-> 我们通过此文章来对其作一个分析对比

File: common/China-Smart-Speaker-Ecosystem-Investigation.md [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+
+<div align="center">
+<img src="files/CM-Smart-Speaker/xiao_du.png">
+</div>
+<div align="center">
+<img src="files/CM-Smart-Speaker/tmall_genie.jpg">
+</div>
+<div align="center">
+<img src="files/CM-Smart-Speaker/xiao_ai.png">
+</div>
```
Comment (ChengYuan-CY) [inline common/China-Smart-Speaker-Ecosystem-Investigation.md:58]: 这三张图请调换一下顺序，改为tmall_genie, xiaoai, xiaodu

File: common/Alexa-control-the-Bluetooth-Mesh-devices-via-cloud.md [hunk 4]
```diff
@@ -0,0 +1,1031 @@
...
+
+8.	Click the icon of the lambda function for Configuration
+
+<div align="center">
+  <img src="./files/CM-Smart-Speaker/lambda_function_configuration.png">
+</div>  
+</br>
+
+Regarding to the lambda function, we take the *Alexa Smart Home skill sample* released by Amazon as starting point, you can access it from the repository [skill-sample-python-smarthome-switch](https://github.com/alexa/skill-sample-python-smarthome-switch). 
+And also you can just reuse the lambda function we have implemented for this project, the zip package ***alexa-control-btmesh-lambda.zip*** is included in the sub directory and also you can get it from the [github](https://github.com/ChengYuan-CY/Alexa-Control-Bluetooth-Mesh-Devices/tree/master/lambda_function).
```
Comment (ChengYuan-CY) [inline common/Alexa-control-the-Bluetooth-Mesh-devices-via-cloud.md:775]: Please upload the alexa-control-btmesh-lambda.zip to the master branch, and modify the GitHub link as the correct one.

File: common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+
+</details>
+
+## 1. 介绍
+
+IoT物联网变得越来越火热, 智能家居已经走入家庭. 智能设备,如灯, 开关, 空调, 温湿度传感器, 风扇, 扫地机器人都可以通过智能音箱来控制. 极大方便了我们日常的生活和智能家居控制管理. 针对IoT市场, Alibaba提供了完善的生态系统, 包括[AliOS Things](https://github.com/alibaba/AliOS-Things)针对终端设备, 含有Link Kit SDK 来连接阿里云IoT物联网平台-[智能生活开放平台](https://living.aliyun.com/). 同时允许天猫精灵无缝连接, 通过语音来控制智能设备. 在本文, 我们利用ESP32的平台来跑AliOS, 连接到阿里云IoT物联网平台, 通过阿里巴巴的云智能App来控制灯的开/关, 然后绑定天猫精灵,使得我 们能够通过语音来控制灯的开/关和了解灯的状态.
+
+## 2. 安装AliOS Things Studio
+
+阿里巴巴提供了详细的文档介绍如何安装开发环境, 下载AliOS Things的代码, 编译示例代码, 下载代码到目标板. ESP32 devkitc是AliOS Things 支持一款开发板. 详细介绍可以参看此网址&nbsp;[AliOS Things Studio](https://github.com/alibaba/AliOS-Things/wiki/AliOS-Things-Studio).
```
Comment (MarkDing) [inline common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md:40]: `The unnecessary HTML tag can be changed, like &nbsp -> space, &quot->"`

File: common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+*   [人机交互](https://living.aliyun.com/doc?spm=a2c9o.12549863.0.0.4e9d38e48XXAqy#swz51p.html), 这一步可以下载公版App, 并可生成产品配网二维码, 用App扫描此二维码即可添加设备.
+*   [产品发布](https://living.aliyun.com/doc?spm=a2c9o.12549863.0.0.4e9d38e48XXAqy#gs40p8.html), 在这一步可以发布产品, 购买激活码并分配给产品, 我们就可以在App里面找到我们自定义的产品.在这里点击量产管理, 选择量产记录, 找到你已经发布的设备,点击查看, 在弹出窗口下载激活码, 在一个excel文件里包含了Product key, Device Name 和 Device Secret.
+
+## 6. 定制linkkitapp的代码
+
+### a. app\example\linkkitapp\linkkit_example_solo.c     
+
+i. 用生活物联网平台得到的产品信息更新如下宏定义.   
+
+<pre>#<span class="pl-k" style="color: rgb(215,58,73);">define</span> <span class="pl-en" style="color: rgb(111,66,193);">PRODUCT_KEY</span> <span class="pl-s" style="color: rgb(3,47,98);"><span class="pl-pds">&quot;</span>a1X2bEnP82z<span class="pl-pds">&quot;</span></span>
```
Comment (MarkDing) [inline common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md:65]:
````c
#define PRODUCT_KEY "a1X2bEnP82z"
#define PRODUCT_SECRET "7jluWm1zql7bt8qK"
#define DEVICE_NAME "test_06"
#define DEVICE_SECRET "wQ1xOzFH3kLdjCTLfi8Xbw4otRz0lHoq"
````

Comment (MarkDing) [reply common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md:65]: All of source code related text should be changed to mark down format

File: common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md [hunk 7]
```diff
@@ -0,0 +1,448 @@
...
+*   配网成功, 云智能App显现设备列表, 点击设备进入控制界面, 可以控制设备的灯的on/off
+
+## 8. 天猫精灵语音控制灯
+
+*   安装天猫精灵App, 并用淘宝账户登陆
+*   在云智能App中选择&quot;我的&quot;-&gt;&quot;第三方服务&quot;, 选择天猫精灵. 点击绑定账号, 登陆同一个淘宝账号. 灯设备出现在可控设备列表里面.
+*   在天猫精灵App里面, 找到你的灯, 分配好名字, 如书房的灯.
+*   对天猫精灵说&quot;打开书房的灯&quot;, 灯开; &quot;关闭书房的灯&quot;, 灯灭; &quot;现在灯是开的么?&quot;, 回答&quot;现在灯处于关闭状态&quot;
+
+</ac:rich-text-body></ac:structured-macro><ac:structured-macro ac:name="panel" ac:schema-version="1" ac:macro-id="43bffa8a-232f-40fd-80e9-617f530a5644"><ac:rich-text-body>
```
Comment (MarkDing) [inline common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md:143]: Here is the wrong display

File: common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md [hunk 8]
```diff
@@ -0,0 +1,448 @@
...
+*   安装天猫精灵App, 并用淘宝账户登陆
+*   在云智能App中选择&quot;我的&quot;-&gt;&quot;第三方服务&quot;, 选择天猫精灵. 点击绑定账号, 登陆同一个淘宝账号. 灯设备出现在可控设备列表里面.
+*   在天猫精灵App里面, 找到你的灯, 分配好名字, 如书房的灯.
+*   对天猫精灵说&quot;打开书房的灯&quot;, 灯开; &quot;关闭书房的灯&quot;, 灯灭; &quot;现在灯是开的么?&quot;, 回答&quot;现在灯处于关闭状态&quot;
+
+</ac:rich-text-body></ac:structured-macro><ac:structured-macro ac:name="panel" ac:schema-version="1" ac:macro-id="43bffa8a-232f-40fd-80e9-617f530a5644"><ac:rich-text-body>  
+
+***
+
+# 天猫精灵控制_Zigbee_设备
```
Comment (MarkDing) [inline common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md:147]: Get source MD file from https://github.com/MarkDing/ZigbeeBridge-Tmall

File: common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md [hunk 9]
```diff
@@ -0,0 +1,448 @@
+# 天猫精灵对接AliOS 设备
```
Comment (MarkDing) [inline common/Tmall-Genie-control-the-Zigbee-devices-via-cloud-with-AliOS.md:1]: Get source MD file from https://github.com/MarkDing/TmallGenieControlsESP32

File: common/Tmall-Genie-Smart-Speaker-ecosystem-Introduction-and-Setup.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
+Silicon Labs is the leader in silicon, software, and solutions for a smarter, more connected world. And there are lots of our customer are focusing on the smart home marketing, and if we can provide some example projects to extend the capability of our wireless solutions to support Tmall Genie smart speaker, the customers will benefit from the examples for getting to market much faster.  
+
+<div align="center">
+<img src="files/CM-Smart-Speaker/Tmall BlockDigram.png">
+</div>
+
+# Project Description
+
+The [Aligenie](https://www.aligenie.com/)</span>&nbsp;platform&nbsp;provides&nbsp;different&nbsp;solutions&nbsp;that&nbsp;developer&nbsp;can&nbsp;adopt&nbsp;to&nbsp;access&nbsp;the&nbsp;service for controlling smart devices via voice command.
```
Comment (MarkDing) [inline common/Tmall-Genie-Smart-Speaker-ecosystem-Introduction-and-Setup.md:13]: Change unnecessary HTML tag to real ASCII characters.

File: common/Cost-of-Development-Analyses-and-Ecosystem-Comparisons.md [hunk 11]
```diff
@@ -0,0 +1,307 @@
...
+<tr>
+<td colspan="1">Development environment</td>
+<td colspan="1">Very complete</td>
+<td colspan="1">Complete</td>
+<td colspan="1">Good but need license first</td></tr>
+<tr>
+<td colspan="1">Service Type</td>
+<td colspan="1">Clould to Cloud, Locally, Direct to Amazon Cloud, Built-in Alexa</td>
+<td colspan="1">Cloud to Cloud, Locally(In development)</td>
+<td colspan="1">Direct to Apple Cloud</td></tr></tbody></table>
```
Comment (Jim-tech) [inline common/Cost-of-Development-Analyses-and-Ecosystem-Comparisons.md:253]: 这个表格没有编号。

**Pull request merged**